### PR TITLE
feat: added wishlist page

### PR DESCRIPTION
**Pull request by kedark152**
### #feat: added wishlist page
1. added 3 product cards in wishlist page. (wishlist.html & wishlist.css)

**Preview Link:** https://deploy-preview-5--agrostores.netlify.app/pages/wishlist.html

### CODE REVIEW

File: css/page-styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    font-family: var(--lato-font);
+}
+.container{
+    justify-content: center;
+    width: 100%;
+    gap: 1rem;
+    flex-wrap: wrap;
+    
+}
+#heart{
```
Comment (shivampandey0) [inline css/page-styles/wishlist.css]: `id` in CSS is not a good practice. Try to use classes only in CSS

**Review by shivampandey0 — COMMENTED** [review]
Everything looks good.
Added on suggestion.

File: css/page-styles/wishlist.css (hunk 1 above)
Comment (kedark152) [reply css/page-styles/wishlist.css]: Thanks for your Review. I have made these changes in new commit, please check it.

**Review by shivampandey0 — APPROVED** [review]
Suggested changes implemented.
good to merge.

**Review by Namrata-J — APPROVED** [review]

**Pull request merged**